client/modules/IDE/pages/IDEViewMobile.jsx

```diff
@@ -92,18 +91,14 @@ const IDEViewMobile = (props) => {
   return (
     <Screen>
       <Header>
+        <Link to="/" style={{ width: '3rem', marginRight: '1.25rem' }}>
```

Comment:
Since we're using styled components, I think it makes sense to not use any inline CSS like this! You also shouldn't need `aria-label` or `aria-hidden` on the `CloseIcon`, as `aria-label` should be added to the `<Link>`, see https://www.scottohara.me/blog/2019/05/22/contextual-images-svgs-and-a11y.html

Reply:
Makes sense. Just updated this!